python/GangaLHCb/Lib/Applications/BenderBox.py

```diff
@@ -160,7 +160,8 @@ class BenderModule(GaudiExec):
     _schema.version.minor += 0
     ## make entries 
     for key,val in _schema.datadict.iteritems() :
-        if key == 'useGaudiRun'  : val._update( { 'defvalue' : False } )  
+        if key == 'useGaudiRun'  : val._update( { 'defvalue' : False } )
```

Comment:
A bit odd construction to loop over all key,value pairs and then ask for a specific one. Will do nothing in case that `useGaudiRun` doesn't exist, but is that really a good behaviour?